src/Core/Utilities/CoreHelpers.cs

```diff
@@ -798,16 +798,21 @@ public static string ObfuscateEmail(string email)
             return email;
         }
 
-        var sb = new StringBuilder();
```

Comment:
NOTE: I didn't add unit tests for this change as some already existed, but please let me know if you'd prefer a few more